search-app/resources/templates/search-docs-static.html

```diff
@@ -0,0 +1,36 @@
+{% extends "templates/structure.html" %}
+
+{% block title %}{{ site-title }} - {{ page-title }}{% endblock %}
+
+{% block top-nav %}
+<div class="eui-masthead-logo eui-application-logo">
+  <h1><a href="../../">CMR Search</a></h1>
+</div>
+<nav class="main-nav" role="navigation">
```

Comment:
So ... you're right about the footer, but the header/nav had to be tweaked for static sites.

But! Your question got me looking at it again, and we can use the existing base.html template, as long as we set `{{ base-url }}` to `../../` :-)

Much better!